src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -62,6 +62,11 @@ internal abstract class PENamedTypeSymbol : NamedTypeSymbol
         /// </summary>
         private TypeKind _lazyKind;
 
+        /// <summary>
+        /// Lazily initiated HasEmbeddedAttribute
+        /// </summary>
+        private ThreeState _lazyHasEmbeddedAttribute = ThreeState.Unknown;
```

Comment:
yes, this is very uncommon case.

---
In reply to: [113542264](https://github.com/dotnet/roslyn/pull/18715#discussion_r113542264) [](ancestors = 113542264)